projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/collection/read-collection-field.html

```diff
@@ -7,7 +7,7 @@
         <ccd-field-read
           [caseField]="{
             id: caseField.label,
-            label: caseField.label,
+            label: caseField.label | rpxTranslate,
```

Comment:
shouldn't this has been translated in the down level ccd-field-read component? What happened if you had this translated here and then you translate again in ccd-field-read? We should avoid translating in the customized component property but it should be in the html template which can't be broken down anymore.